client/package.json

```diff
@@ -23,5 +23,6 @@
     "eslint-plugin-prettier": "^2.6.2",
     "prettier": "1.14.0",
     "prettier-eslint": "^8.8.2"
-  }
+  },
+  "proxy": "http://localhost:4000"
```

Comment:
This proxies all requests for `/api` to the server. This brings the dev setup closer to what we have in production i.e. if `REACT_APP_API_URI` is not set then the fallback route will still work.